Taller152/src/paqutedos/Principal.java

```diff
@@ -5,11 +5,54 @@
  */
 package paqutedos;
 
+import java.util.ArrayList;
+import java.util.Scanner;
+
 
 /**
  *
  * @author reroes
  */
 public class Principal {
+    public static void main(String args[]) {
+        
+        ArrayList<String> ListaEquipos = new ArrayList<>();
+        Scanner entrada = new Scanner(System.in);
+        String nombres;
+        double presupuestoAnual;
+        String estadio;
+        String cadena = "";
+        int dato;
+        boolean bandera = true;
+    
+    
+        while (bandera) { 
+            
+            System.out.println("Ingrese el nombre del equipo");
+                nombres = entrada.nextLine();
+            System.out.println("Ingrese el presupuesto anual");
+                presupuestoAnual = entrada.nextDouble();
+            System.out.println("Ingrese el nombre del estadio");
+                entrada.nextLine();
+                estadio = entrada.nextLine();
+            System.out.printf("%s\n", "Ingrese el número 2"
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.